server/integration_tests/test_data/e2e_edge_cases/emissionsinhouston/chart_config.json

```diff
@@ -623,57 +623,7 @@
   ],
   "relatedThings": {
     "childPlaces": {},
-    "childTopics": [
```

Comment:
Argh, we might need to check fallback place in certain cases.

https://github.com/datacommonsorg/website/blob/a3aaccfc8236dc9eaa82a2e46baaa13b834cbd2a/server/routes/nl/helpers.py#L187-L193

If you want to defer to me that's OK.